REST/RESTListener+Replicate.cc

```diff
@@ -77,7 +77,21 @@ namespace litecore { namespace REST {
                     ((ReplicationTask*)context)->onReplStateChanged(status);
                 };
                 params.callbackContext = this;
-
+                if (_authType) {
```

Comment:
See other comment about auth type.

Reply:
Got it, will test it against "Basic". What to do with the else part: ignore and log a runtime warning?

Reply:
Return a 401 status.